### ISSUE TITLE
image comments viewing attempt error

### ISSUE THREAD

**Issue by kevinmoilar** [nextcloud/android#12600]
### ⚠️ Before posting ⚠️

- [X] This is a **bug**, not a question or an enhancement.
- [X] I've [searched for similar issues](https://github.com/nextcloud/android/issues) and didn't find a duplicate.
- [X] I've written a clear and descriptive title for this issue, not just "Bug" or "Crash".
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Steps to reproduce

1. add a commebt to an image in a laptop / desktop webbrowser
2. find comment next to image in list view and click it

### Expected behaviour

show comment

### Actual behaviour

error thrown:
### Cause of error
```java
Exception in thread "Thread-18" java.lang.IllegalArgumentException: Trying to execute a remote operation with a NULL OwnCloudClient
    at com.owncloud.android.lib.common.operations.RemoteOperation.execute(RemoteOperation.java:201)
    at com.owncloud.android.ui.fragment.FileDetailActivitiesFragment.lambda$markCommentsAsRead$7(FileDetailActivitiesFragment.java:372)
    at com.owncloud.android.ui.fragment.FileDetailActivitiesFragment.$r8$lambda$YWBrXKUAdoytvoaiG4-M3S75PWU(Unknown Source:0)
    at com.owncloud.android.ui.fragment.FileDetailActivitiesFragment$$ExternalSyntheticLambda8.run(Unknown Source:2)
    at java.lang.Thread.run(Thread.java:1012)

```

### App information
* ID: `com.nextcloud.client`
* Version: `30280090`
* Build flavor: `gplay`

### Device information
* Brand: `Fairphone`
* Device: `FP5`
* Model: `FP5`
* Id: `TKQ1.230127.002`
* Product: `FP5`

### Firmware
* SDK: `33`
* Release: `13`
* Incremental: `TT45`

### Android version

13

### Device brand and model

fairphone 5

### Stock or custom OS?

Stock

### Nextcloud android app version

30280090

### Nextcloud server version

27.1.6

### Using a reverse proxy?

No

**Comment by kevinmoilar** [comment]
might be related to a server misconfiguration. will check and close if so.

**Comment by joshtrichards** [comment]
Fixed in #12583
